iOS Example Frame/iOS Example Frame/Style/OtherStyle.swift

```diff
@@ -0,0 +1,274 @@
+//
+//  OtherStyle.swift
+//  iOS Example Frame
+//
+//  Created by Alex Ioja-Yang on 18/08/2022.
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+import UIKit
+import Frames
+
+enum OtherStyle {
```

Comment:
OtherStyle might make sense when looking at it from the application buttons, but it does not make sense when thinking "why it exists".

ThemeDemo was here to show how Theme can be used to achieve custom UI in a simple demonstrative way. Renaming it to OtherStyle doesn't help for its purpose as a way to demonstrate anything.